docs/from_bosh_to_kube.md

```diff
@@ -335,6 +345,54 @@ tags:
 
 ### Healthchecks
 
+## Conversion Details
+
+### Calculation of docker image location for releases
+
+### Variables to Extended Secrets
+
+For each **BOSH Variable**, the cf-operator creates `ExtendedSecrets`.
```

Comment:
I don't understand the need to store BOSH variables individually; they could all be stored in a single secret per deployment.

Reply:
They're all created individually using `ExtendedSecrets`.
The size of a large secret could likely go over the limit.